data/core/units/saurians/Soothsayer.cfg

```diff
@@ -8,11 +8,10 @@
     hitpoints=29
     movement_type=lizard
     movement=6
-    experience=100
+    experience=60
```

Comment:
This experience seems low to me.

Reply:
@nemaara Saurians advance at a faster rate than other units. To put it in some context:
* Skirmisher→Ambusher: 32
* Augur→Soothsayer: 28
* Ambusher→Flanker: 55
* **Soothsayer→Seer: 60**
* Mage→Red Mage: 54
* Red Mage→Arch Mage: 100

Saurian mages require roughly half of the experience of the human mages, while maintaining a less offensive healer role. The experience rates were also discussed with @Hejnewar, but please propose a specific value if you have one in mind.

Reply:
I meant in terms of the other advancement is 66 and I would say that this healer probably deserves more exp than that. I also agree on the saurian healers needing less exp compared to other mages so maybe that one can come down to 60 or even less and I would be fine with it.

Reply:
Oracle is stronger offensively, hence easier to advance in general. That's mostly why the Seer requires less experience. I think the effort required to advance Soothsayer to Seer and Oracle to Prophet is pretty comparable, although of course Soothsayer has more utility as a healer.

Reply:
Dont mind XP too much, in wesnoth at this point it can be quiet random anyway and some day I will probably try to fix that myself but it is a lot of work so it is hard to say when exactly.

Reply:
Okay, we can just leave the exp for now then.